src/components/manage/Widgets/story.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
```

Comment:
@tiberiuichim only a last thing: Isn't it this name too generic? ;) Also, shouldn't the file name be Capitalised?

Reply:
@sneridagh I thought to keep only the Widget modules upper-cased, otherwise people reading Volto's source code will open the story utils file when it's not actually interesting. The name could be changed, maybe storybook.jsx ? storyutils?